File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;

    let pCount = 0
    let yCount = 0
    
    for(let e of s){
        if(e==="P" || e === "p")  ++pCount
    else if (e=== "y" || e === "Y") ++yCount   
    }
   
    answer = pCount === yCount ? true : false
    
    return answer;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    countP = 0
    countY = 0
    
    for i in s :
        if(i.lower() == "p"):
            countP+=1
        elif(i.lower() == 'y'):
            countY+=1
    
    
    
    

    return True if countP == countY else False